showcase/scripts/generate_showcase.sh

```diff
@@ -1,129 +1,95 @@
-  mkdir "${well_known_folder}"
-fi
-if [[ -f "${well_known_generator_jar_location}" ]]; then
-  echo "replacing well-known generator jar with the latest one"
-  rm "${well_known_generator_jar_location}"
-fi
-if [[ -f "${well_known_formatter_jar_location}" ]]; then
-  echo "replacing well-known formatter jar with the latest one"
-  rm "${well_known_formatter_jar_location}"
-fi
-maven_repository="$(mvn help:evaluate -Dexpression=settings.localRepository -q -DforceStdout)"
-generator_version=$(get_version_from_versions_txt "gapic-generator-java")
-generator_jar_path="${maven_repository}/com/google/api/gapic-generator-java/${generator_version}/gapic-generator-java-${generator_version}.jar"
+GENERATOR_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -pl gapic-generator-java)
 
-if [[ ! -f "${generator_jar_path}" ]]; then
-  echo "generator jar not found in its assumed location"
-  echo "in the local repository: ${generator_jar_path}"
-  echo "(did you run mvn install in this repository's root?)"
-  exit 1
-fi
-# transfer the snapshot jar into its well-known location
-cp "${generator_jar_path}" "${well_known_generator_jar_location}"
-# transfer java formatter to its well-known location
-download_from \
-  "https://maven-central.storage-download.googleapis.com/maven2/com/google/googlejavaformat/google-java-format/${java_formatter_version}/google-java-format-${java_formatter_version}-all-deps.jar" \
-  "${java_formatter_name}"
-cp "${java_formatter_name}" "${well_known_formatter_jar_location}"
-gapic_additional_protos="google/iam/v1/iam_policy.proto google/cloud/location/locations.proto"
-
-path_to_generator_parent_pom="${SCRIPT_DIR}/../../gapic-generator-java-pom-parent/pom.xml"
-protoc_version=$(get_version_from_pom "${path_to_generator_parent_pom}" "protobuf.version" \
-  | cut -d. -f2-)
-download_protoc "${protoc_version}" "linux-x86_64"
-mv "bin" "include" "${well_known_folder}"
-grpc_version=$(get_version_from_pom "${path_to_generator_parent_pom}" "grpc.version")
-download_grpc_plugin "${grpc_version}" "linux-x86_64"
-mv "protoc-gen-grpc-java.exe" "${well_known_folder}"
-rest_numeric_enums="false"
-transport="grpc+rest"
-gapic_yaml=""
-service_config="schema/google/showcase/v1beta1/showcase_grpc_service_config.json"
-service_yaml="schema/google/showcase/v1beta1/showcase_v1beta1.yaml"
-include_samples="false"
-rm -rdf output/showcase-output
-mkdir output/showcase-output
-set +e
-bash "${SCRIPT_DIR}/../../hermetic_build/library_generation/generate_library.sh" \
-  --proto_path "schema/google/showcase/v1beta1" \
-  --destination_path "showcase-output" \
-  --gapic_additional_protos "${gapic_additional_protos}" \
-  --rest_numeric_enums "${rest_numeric_enums}" \
-  --gapic_yaml "${gapic_yaml}" \
-  --service_config "${service_config}" \
-  --service_yaml "${service_yaml}" \
-  --include_samples "${include_samples}" \
-  --transport "${transport}" 
+echo "generating showcase"
+workspace_name="/workspace"
+docker run \
```

Comment:
Since this is using docker now, I don't think we can run them from local Mac or Windows anymore, can you please update the README of the showcase module to indicate it?

Reply:
Good point. Done.